### ISSUE TITLE
Training YOLONAS from scratch for using it in a commercial application

### ISSUE THREAD

**Issue by albertofernandezvillan** [Deci-AI/super-gradients#901]
As far as I understand for the license, if I used pre-trained weights for training YOLONAS (fine-tune my model on my dataset), I cannot use it in commercial applications. Is it right?

If so, when training:

```
from super_gradients.training import models
model = models.get('yolo_nas_l', 
                   num_classes=len(dataset_params['classes']), 
                   pretrained_weights="coco"
                   )
```
how to change this and train it from scratch?

**Comment by NatanBagrov** [comment]
Hello, to get a model without pre-trained weights, you should remove the relevant part (`pretrained_weights="coco"`). This is applicable to any model.

**Comment by albertofernandezvillan** [comment]
Thanks, and what about "I used pre-trained weights for training YOLONAS (fine-tune my model on my dataset), I cannot use it in commercial applications. Is it right?"

**Comment by NeovMs** [comment]
If we use the pre-trained models for knowledge distillation to pre-train the Yolo-NAS from scratch, we're not strictly using the weights. In this case, can we get round the licence ?
